cabal.project

```diff
@@ -58,9 +58,18 @@ package plutus-scripts-bench
   haddock-options: "--optghc=-fplugin-opt PlutusTx.Plugin:defer-errors"
 
 constraints:
-  wai-extra < 3.1.15,
+  , wai-extra < 3.1.15
+  , plutus-ledger-api ^>= 1.31
+  , ouroboros-network ^>= 0.16.1
```

Comment:
Who do you need these two? Aren't those picked up just by cabal picking the latest version?

Reply:
Removed.